constantinCurtis/src/main/java/data/collection/task/Main.java

```diff
@@ -18,19 +17,23 @@ public static void main(String[] args) {
 
         //create items
         List<Item> ItemFirstList = new ArrayList<>();
-        Item itemJeans = new Item("Jeans", Type.CLOTHES);
-        Item itemBall = new Item("Ball", Type.TOY);
+        Item itemJeans = new Item("Jeans", Type.CLOTHES, 27.1);
+        Item itemBall = new Item("Ball", Type.TOY, 28.5);
         //create second Item List
 
         List<Item> ItemSecondList = new ArrayList<>();
-        Item itemSamsung = new Item("Samsung", Type.GADJET);
-        Item itemJacket = new Item("Jacket", Type.CLOTHES);
+        Item itemSamsung = new Item("Samsung", Type.GADGET, 28.7);
+        Item itemJacket = new Item("Jacket", Type.CLOTHES, 35.4);
 
         // adding items to ItemFinal List
         ItemFirstList.add(itemJeans);
         ItemFirstList.add(itemBall);
         ItemSecondList.add(itemSamsung);
         ItemSecondList.add(itemJacket);
+        ItemSecondList.add(itemJacket);
+        AbstractOrder abstractOrder = new AbstractOrder(ItemFirstList) {
```

Comment:
Why you create an object from AbstractOrder class which is abstract?
Not a good practice.

Reply:
Thank you, changed.